src/containers/WorkspaceArea.js

```diff
@@ -24,6 +26,103 @@ const mapStateToProps = state => (
  * @returns {{background: {background: string}}}
  */
 const styles = theme => ({
+  '@global': {
+    '.mirador-canvas-nav': {
+      display: 'flex',
+      flexDirection: 'row',
+      flexWrap: 'wrap',
+      justifyContent: 'center',
+      textAlign: 'center',
+    },
+    '.mirador-canvas-nav-stacked': {
+      flexDirection: 'column',
+    },
+    '.mirador-companion-window-bottom': {
+      '&.mirador-companion-window-title-controls': {
+        order: 'unset',
+      },
+    },
+    '.mirador-companion-window-header': {
+      flexWrap: 'wrap',
+    },
+    '.mirador-companion-window-title-controls': {
+      flexGrow: 1,
+      order: 1000,
+    },
+    '.mirador-label-value-metadata dd': {
+      marginBottom: '.5em',
+      marginLeft: 0,
+    },
+    '.mirador-osd-container': {
+      flex: 1,
+      position: 'relative',
+    },
+    '.mirador-osd-info': {
+      order: 2,
+      overflow: 'hidden',
+      paddingBottom: 3,
+      textOverflow: 'ellipsis',
+      whiteSpace: 'nowrap',
+      width: '100%',
+    },
+    '.mirador-osd-navigation': {
+      order: 1,
+    },
+    '.mirador-primary-window': {
+      display: 'flex',
+      flex: 1,
+      position: 'relative',
+    },
+    '.mirador-viewer': {
+      bottom: 0,
+      left: 0,
+      overflow: 'hidden',
+      position: 'absolute',
+      right: 0,
+      top: 0,
+    },
+    '.mirador-workspace': {
+      boxSizing: 'border-box',
+      margin: 0,
+      position: 'absolute',
+      transitionDuration: '.7s',
+      '&.react-draggable-dragging': { // eslint-disable-line sort-keys
+        transitionDuration: 'unset',
+      },
+    },
+    '.mirador-workspace-add': {
+      height: '100%',
+      overflowX: 'hidden',
+      overflowY: 'auto',
+      paddingTop: 68,
+      [theme.breakpoints.up('sm')]: {
+        paddingLeft: 68,
+        paddingTop: 0,
+      },
+    },
+    '.mirador-workspace-with-control-panel': {
+      paddingTop: 74,
+      [theme.breakpoints.up('sm')]: {
+        paddingLeft: 68,
+        paddingTop: 0,
+      },
+    },
+    '.mosaic-tile': {
+      boxShadow: [
```

Comment:
does not serialize

Reply:
Resolved in latest commit. Seemed to be a case of merge ordering.